diktat-maven-plugin/src/main/kotlin/org/cqfn/diktat/plugin/maven/DiktatMojo.kt

```diff
@@ -38,12 +38,11 @@ class DiktatFixMojo : DiktatBaseMojo() {
     @OptIn(FeatureInAlphaState::class)
     override fun runAction(params: KtLint.ExperimentalParams) {
         val fileName = params.fileName
-        val filePath = params.userData["file_path"] ?: error("File path should be provided")
-        val fileContent = File(filePath).readText(charset("UTF-8"))
+        val fileContent = File(fileName).readText(charset("UTF-8"))
         val formattedText = KtLint.format(params)
         if (fileContent != formattedText) {
             log.info("Original and formatted content differ, writing to $fileName...")
-            File(filePath).writeText(formattedText, charset("UTF-8"))
+            File(fileName).writeText(formattedText, charset("UTF-8"))
```

Comment:
now we print a full path.
@petertrr @akuleshov7 is it okay for us?

Reply:
Without access to the full file path we can't perform some checks, e.g. whether package name matches file location. Here - I'm surprised that it works, because I'm pretty sure this `File` will be relative to maven's working directory, which is project root and not source location

Reply:
it was relative to working directory before, after latest changes -- it's a full path (absolute)